stable/osg-hosted-ce/osg-hosted-ce/values.yaml

```
@@ -176,7 +176,12 @@ HTTPLogger:
 
 # If a specific node is desired for scheduling
 # set this to the hostname of the desired node
-NodeSelection: null
+NodeSelection:
+  Enabled: false
+  Hostname: null
+  # Enter as many desired label key value pairs
+  NodeLabels:
+    - example: label
```

```suggestion
  # Enter as many desired label key value selector pairs
  # NodeLabels:
  #  For example, to specify a particular Kubernetes host for the deployment:
  #  - kubernetes.io/hostname: example.host.org
```

1. Any values in the default `values.yaml` will get applied to deployments so this would result in `example: label` showing up in the `nodeSelector` and since that's not a legit selector, it'll probably blow up existing deployments so you'll want to comment it out
2. May as well provide an example of the hostname selector here